pkg/asset/cluster/cluster.go

```diff
@@ -49,6 +49,7 @@ func (c *Cluster) Dependencies() []asset.Asset {
 	return []asset.Asset{
 		&installconfig.ClusterID{},
 		&installconfig.InstallConfig{},
+		&installconfig.PlatformCredsCheck{},
```

Comment:
Can you add a note here for why this dependency is needed? Since the dependency is not fetched in `Generate`, it may be removed accidentally here later without a note explaining why it is indeed needed.

Reply:
I was confused about this too, but given the leading comment, I'm guessing this is giving folks a last chance to recover before we head to Terraform and die because we don't have credentials setup.